test/functional/Java22andUp/src/org/openj9/test/jep454/downcall/PrimitiveTypeTests1.java

```diff
@@ -263,7 +280,7 @@ public void test_strlenFromDefaultLibWithMemAddr_1() throws Throwable {
 		MemorySegment strlenSymbol = defaultLibLookup.find("strlen").get();
 		FunctionDescriptor fd = FunctionDescriptor.of(JAVA_LONG, ADDRESS);
 		MethodHandle mh = linker.downcallHandle(strlenSymbol, fd);
-		MemorySegment funcSegmt = arena.allocateFrom("JEP454 DOWNCALL TEST SUITES");
+		MemorySegment funcSegmt = allocateString("JEP424 DOWNCALL TEST SUITES");
```

Comment:
I don't think the string needs to change (it can still refer to JEP 454).

Reply:
Fixed everywhere.